components/calculator.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react'
+
+export default () => {
+  const [answer, setAnswer] = useState('')
+  const [error, setError] = useState('')
+  const [numOne, setNumOne] = useState('')
+  const [numTwo, setNumTwo] = useState('')
+  const [operator, setOperator] = useState('')
+
+  const calculateAndUpdateAnswer = () => {
+    const num1 = Number(numOne)
+    const num2 = Number(numTwo)
+    if (num1 && num2) {
+      if (error) setError('')
+      switch (operator) {
+        case 'add':
+          setAnswer(num1 + num2)
+          break
+        case 'subtract':
+          setAnswer(num1 - num2)
+          break
+        case 'multiply':
+          setAnswer(num1 * num2)
+          break
+        case 'divide':
+          setAnswer(num1 / num2)
+          break
+        default:
+          setAnswer(num1 + num2)
+      }
+    } else {
+      setError('something went wrong.')
+      setAnswer('')
+    }
+  }
+  return (
+    <div className="page">
+      <h1>React Calculator</h1>
+      <input type="text" onChange={(e) => { setNumOne(e.target.value) }} />
+      <select className="operator" defaultValue="add" onChange={(e) => { setOperator(e.target.value) }}>
+        <option value="add">+</option>
+        <option value="subtract">-</option>
+        <option value="multiply">*</option>
+        <option value="devide"> / </option>
```

Comment:
This typo here means that you add when you want to divide

![image](https://user-images.githubusercontent.com/5775716/83040098-e3325600-a00c-11ea-921c-cfb014fe4944.png)